larpandoracontent/LArHelpers/LArGeometryHelper.h

```diff
@@ -208,6 +208,20 @@ class LArGeometryHelper
     static bool IsInGap(const pandora::Pandora &pandora, const pandora::CartesianVector &testPoint2D, const pandora::HitType hitType,
         const float gapTolerance = 0.f);
 
+    /**
+     *  @brief  Whether a pair of 2D test points lies in the same registered gap with the associated hit type
+     *
+     *  @param  pandora the associated pandora instance
+     *  @param  p1 the first test point
+     *  @param  p2 the second test point
+     *  @param  hitType the hit type
+     *  @param  gapTolerance the gap tolerance
+     *
+     *  @return boolean
+     */
+    static bool IsInGap(const pandora::Pandora &pandora, const pandora::CartesianVector &p1, const pandora::CartesianVector &p2,
```

Comment:
Just a thought - should the function name make it clear that the requirement is for the points to live in the *same* gap? Something like `IsInSingleGap`, `DoPointsLieInSingleGap`, etc.? I'm sure you can think of a better name than these two suggestions!